src/components/CellsContainer/CellsContainer.svelte

```diff
@@ -0,0 +1,201 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  import { select } from 'd3';
+  import { createSvg, getId, createRectsData, setAttributes } from './helpers';
+
+  export let count = 2;
+  export let size = 100;
+  export let gap = 1;
+  export let getAdditionalRectProps;
```

Comment:
I wouldn't pass a function as input param. The only role which I can expect from functions in the props is a some component event callback

Reply:
We need to get additional cell props based on index and liveCount
Other way I see: transfer liveCount to props and use getCellProps right there.
And createRectsData function is pure. It uses only getRectX and getRectY and they are pure.

Reply:
Okay, that can make sense